### PR TITLE
Minor. Updated README and added publish.sh from apache/bahir-website

### PR DESCRIPTION
**Pull request by ajbozarth**
I've updated the README based on [apache/bahir-website](https://github.com/apache/bahir-website) and included `publish.sh` from that repo based on feedback in [INFRA-14659](https://issues.apache.org/jira/browse/INFRA-14659)

### CODE REVIEW

**Comment by jerryshao** [comment]
@ajbozarth , what is the usage of publish.sh?

**Comment by jerryshao** [comment]
I don't think it is mergeable directly from Github, let me push a merge script first to this repo.

**Comment by ajbozarth** [comment]
From what I understand the live website code must be stored in the `asf-site` branch and the script (which is taken directly from the bahir-website repo) does all the work of updating that branch from the latest master so it can be picked up by the automated deploy

File: README.md [hunk 1]
```diff
@@ -17,67 +17,58 @@ limitations under the License.
...
+[livy.incubator.apache.org](https://livy.incubator.apache.org/).
 
 ## Adding contributors
 
-To add a contributor to the project, or to modify existing contributors,
-edit `site/_data/contributors.yml`.
-The [project members]([http://localhost:4000/community-members/]
-list will re-generate.
+To add a contributor to the project, or to modify existing contributors, edit `site/_data/contributors.yml`.
+The [project members](http://localhost:4000/community-members/) list will re-generate.
```
Comment (jerryshao) [inline README.md:74]: Shall we update this link? Currently it is pointing to localhost.

Comment (ajbozarth) [reply README.md:74]: I left it at localhost since this section is aimed at local development.

**Comment by lresende** [comment]
publish.sh get the contents of master and update to the asf-site branch which, after manual push will update the contents of the live website. The manual push is mainly for enabling a final content review before updating the website.

**Comment by jerryshao** [comment]
Thanks for the explanation, LGTM.

**Comment by ajbozarth** [comment]
now this is merged can you run the publish script?

**Comment by lresende** [comment]
Wait, we need to create an orphan branch first, have we done that yet?

**Comment by jerryshao** [comment]
@lresende can you please handle this, I'm not familiar with whole publish process.

Also do you have some guidance for me to refer. Thanks!

**Comment by lresende** [comment]
All done. Mostly you create an orphan branch to avoid carrying the history from the website source
```
git checkout --orphan asf-site
```
then add some file and push.... 
now, after that, we can always just use the publish.sh utility script

**Pull request closed without merging**